Code/PgSQL/rdkit/adapter.cpp

```diff
@@ -2173,7 +2173,7 @@ extern "C" char *findMCSsmiles(char *smiles, char *params) {
   while (*s && *s <= ' ') {
     s++;
   }
-  while (s < s_end && *s > ' ') {
+  while (s<s_end && * s> ' ') {
```

Comment:
This formatting change looks strange, so I’d revert it:
```suggestion
  while (s < s_end && *s > ' ') {
```